pkg/workload/workload.go

```diff
@@ -893,6 +904,34 @@ func HasConditionWithTypeAndReason(w *kueue.Workload, cond *metav1.Condition) bo
 	return false
 }
 
+func HasFailedNodeAnnotation(w *kueue.Workload) bool {
+	if w == nil {
+		return false
+	}
+	annotations := w.GetAnnotations()
+	_, hasFailedNodeAnnotation := annotations[kueuealpha.NodeToReplaceAnnotation]
```

Comment:
```suggestion
	_, found := annotations[kueuealpha.NodeToReplaceAnnotation]
```
nit, this is a short-living var name so it is preferred to make it short in golang

Reply:
Done